app/controllers/ApplicationController.scala

```diff
@@ -27,14 +27,14 @@ class ApplicationController @Inject() (
    *
    * @return The result to display.
    */
-  def user = SecuredAction.async { implicit request =>
+    def user : Action[AnyContent] = SecuredAction.async { implicit request =>
```

Comment:
Research for conversion did not yield a result; it seems that Ok(…) always yields AnyContent, even if JsValue is passed in like here). Nonetheless I'll do some more research to find out if changing this is possible.